src/main/groovy/cruise/umple/UmpleGradlePlugin.groovy

```diff
@@ -9,34 +9,62 @@ import org.gradle.api.GradleException
 
 class UmpleGradlePlugin implements Plugin<Project> {
 
-	private UmpleConsoleConfig consoleConfig 
-	private UmpleConsoleMain consoleMain
+	// Member variables
+	private String m_umpleFileName = ""
+	private String m_languageToGenerate = ""
+	private String m_outputPath = ""
+	private String m_defaultOutputPath = "../../libs/"
```

Comment:
The idea was to have a common place to store the files. The 'correct' way is with mavenLocal(), but from what I gathered that requires a Maven repo to be setup on the machine and an environment variable. At the time, it didn't seem wise to add that dependency/complexity.

For this one in particular I've changed it to '/umpleOutput/', which is technically still relative, but I'm not sure what you'd like me to do. The reason I was creating a directory up one level was since both this gradle plugin repo and my gradle plugin are in the same directory, they can both go up one and share the same files. It seemed somewhat more efficient than having both download the umple jar for example.

If '/umpleOutput/' is not what you would like it to default to, what should I change it to?